frame/staking/src/lib.rs

```diff
@@ -564,8 +617,12 @@ pub trait SessionInterface<AccountId>: frame_system::Trait {
 	fn disable_validator(validator: &AccountId) -> Result<bool, ()>;
 	/// Get the validators from session.
 	fn validators() -> Vec<AccountId>;
+	/// Get the validators and their corresponding keys from session.
+	fn keys<KeyType: RuntimeAppPublic + Decode + sp_std::fmt::Debug>() -> Vec<(AccountId, Option<KeyType>)>;
```

Comment:
Is it cool to make changes here? specially this function made me feel quite guilty..